tests/integ/sagemaker/lineage/test_endpoint_context.py

```diff
@@ -24,11 +27,22 @@ def test_model(
     model_list = endpoint_context_associate_with_model.models()
     for model in model_list:
         assert model.source_arn == endpoint_action_obj.action_arn
-        assert model.destination_arn == model_obj.context_arn
+        assert model.destination_arn == model_obj.artifact_arn
         assert model.source_type == "ModelDeployment"
         assert model.destination_type == "Model"
 
 
+def test_model_v2(endpoint_context_associate_with_model, model_obj, sagemaker_session):
+    time.sleep(SLEEP_TIME_ONE_SECONDS)
```

Comment:
What's this for?

Reply:
Looks like the lineage ingestion take a while, sometimes I get 0 length for the query lineage. So I add 1 second sleep here.

Reply:
> 

Looks like the lineage ingestion take a while, sometimes I get 0 length for the query lineage. So I add 1 second sleep here.